unfetter-discover/unfetter-ui
# stub out summary results for Capabilities assessments only

supports unfetter-discover/unfetter#1084

To Test
Go to Assess Beta
Check that all types of assessments other than capability type display data in all charts and graphs
Check that a capability type assessment displays "Implementation COMING SOON" in all blocks
